mex/extractors/datenkompass/transform.py

```diff
@@ -212,6 +262,115 @@ def transform_bibliographic_resources(
                 ),
                 identifier=item.identifier,
                 entityType=item.entityType,
-            )
+            ),
         )
     return datenkompass_bibliographic_recources
+
+
+def transform_resources(
+    merged_resources_by_primary_source: dict[str, list[MergedResource]],
+    filtered_merged_activities: list[MergedActivity],
+    merged_bmg_ids: set[MergedOrganizationIdentifier],
+    merged_organizational_units_by_id: dict[
+        MergedOrganizationalUnitIdentifier, MergedOrganizationalUnit
+    ],
+    merged_contact_points_by_id: dict[MergedContactPointIdentifier, MergedContactPoint],
+) -> list[DatenkompassResource]:
+    """Transform merged to datenkompass resources.
+
+    Args:
+        merged_resources_by_primary_source: dictionary of merged resources
+        filtered_merged_activities: list of merged activities
+        merged_bmg_ids: set of merged bmg organization identifiers
+        merged_organizational_units_by_id: dict of merged organizational units by id
+        merged_contact_points_by_id: dict of merged contact points
+
+    Returns:
+        list of DatenkompassResource instances.
+    """
+    datenkompass_recources = []
+    merged_activities_set = {
+        ma.identifier
+        for ma in filtered_merged_activities
+        if any(fOC in merged_bmg_ids for fOC in ma.funderOrCommissioner)
+    }
+    for (
+        primary_source,
+        merged_resources_list,
+    ) in merged_resources_by_primary_source.items():
+        for item in merged_resources_list:
+            if item.accessRestriction == AccessRestriction["RESTRICTED"]:
+                voraussetzungen = "Zugang eingeschränkt"
+            elif item.accessRestriction == AccessRestriction["OPEN"]:
+                voraussetzungen = "Frei zugänglich"
+            frequenz = (
+                get_vocabulary([item.accrualPeriodicity])
+                if item.accrualPeriodicity
+                else []
+            )
+            kontakt = get_resource_contact(
+                sorted({*item.contact, *item.unitInCharge}),
+                merged_organizational_units_by_id,
+                merged_contact_points_by_id,
+            )
+            beschreibung = "n/a"
+            if item.description:
+                description_de = [
+                    d.value for d in item.description if d.language == "de"
+                ]
+                beschreibung = (
+                    description_de[0] if description_de else item.description[0].value
+                )
```

Comment:
```suggestion
                beschreibung = (
                    description_de[0] if description_de else item.description[0].value
                )
```
isn't this just the first item of the list above?

Reply:
it's supposed to search for a German description and use that. Otherwise use the first description available.